test/reducers/vedlikeholdTest.js

```diff
@@ -4,52 +4,50 @@ import * as actions from '../../js/actions/vedlikehold_actions';
 import vedlikehold from '../../js/reducers/vedlikehold';
 
 describe('vedlikehold', () => {
-    describe('henter', () => {
-        let initialState = deepFreeze({
-            data: {},
+    let initialState = deepFreeze({
+        data: {},
+        henter: false,
+        hentingFeilet: false,
+    });
+
+    it('håndterer VEDLIKEHOLD_HENTET', () => {
```

Comment:
Når det er laget en funksjon for å generere reducere. Kunne vi testet den og dermet vært sikker på at alle reducere den lager er ok og dermed slippe å teste dem?